src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -265,6 +265,37 @@ describe('NewWorkspaceModal', () => {
     expect(screen.queryByText('Importing directly into new Azure workspaces is not currently supported.')).toBeNull();
   });
 
+  it('hides unprotected Azure billing projects when additional security monitoring is required', async () => {
```

Comment:
Moved this test next to a related test. The following tests this in combination with a specific cloud platform.

Also clarified the intent. It's hiding unprotected Azure billing projects, not all Azure billing projects.